_gsocorgs/2018/diana-hep.md

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
I believe this is for real the last comment 😄 
This file should be a project, not an organization. For an example of how to define and where to place a project's .md file, please see:
https://github.com/HEP-SF/hep-sf.github.io/tree/master/_gsocprojects/2018

Reply:
Done.